Calculator/CalculatorLogic/CalculatorLogic.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.ComponentModel;
+
+namespace CalculatorLogic
+{
+    public class CalculatorLogic
+    {
+        double firstNumber;
+        double secondNumber;
+        public string CurrentOutput { get; set; }
+        char currentOperation;
+        string currentState = "WaitingForFirstNumber";
+        public void Calculate(double number, char operation)
+        {
+            switch (currentState)
+            {
+                case "WaitingForFirstNumber":
+                    {
+                        firstNumber = number;
+                        CurrentOutput = firstNumber.ToString() + " " + operation + " ";
```

Comment:
```suggestion
                        CurrentOutput = $"{firstNumber} {operation} ";
```
Если нужно конкатенировать несколько строк, используй форматную строку.
Так и читаемее, и эффективнее (несколько последовательных конкатенаций порождают промежуточные строки.
Может, компилятор это соптимизирует, а может и нет.